### PR TITLE
add shell script for automated install

### PR DESCRIPTION
**Pull request by circa10a**
Created a shell script for issue #7 , (with lots of error checking) for automated setup, install. Also modified the docs to read commands easier.

Shell script was tested on ubuntu and does everything but install (apt-get, yum)packages, due to security concerns.

### CODE REVIEW

**Review by nikssardana — COMMENTED** [review]
Hey @circa10a 
I will test the script on a new server and check if it works as expected. 
I'll merge it after that.

**Comment by nikssardana** [comment]
@circa10a can you please add a line to chown the database to current user?

**Comment by circa10a** [comment]
I didn't see that in your docs provided, can you give me an example of what command you ran?

**Comment by nikssardana** [comment]
@circa10a Apart from your script, I had to run the following command on my server:
`sudo chown nikhil:nikhil db.sqlite3`
Otherwise, there were issues in logging into the app.
Can we add something like this in your script, so that it detects the current user of the application and changes the ownership of db.sqlite3 to the current user?
Btw, you did really nice work there :)

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+  echo "Failed to create Database, exiting..."
+  $reset
+  exit 1
+fi
+
+if $python $install_dir/gdriveTorrent/gdriveTorrent/manage.py createsuperuser; then
+  $green
+  echo "Created superuser"
+
+  if sudo chown -R $(whoami):$(whoami) $install_dir/ && sudo chmod -R 755 $install_dir/; then
```
Comment (circa10a) [inline install.sh:261]: I've added the `chown` fix here.

File: README.md [hunk 2]
```diff
@@ -10,30 +10,70 @@ A Django Application that allows you to download torrents directly to your Googl
...
+```
+
+Note: (Tested on ubuntu)
+
+apt-get/yum packages are not installed automatically due to security concerns, the installer will inform you of any packages missing (if there are any), but here is a list of packages needed:
+```
+sudo apt-get install python-pip python-dev build-essential aria2 git wget -y && sudo pip install virtualenv
+```
+
+After setup is complete, you may start the server at any time (or have a job do it) by running:
```
Comment (circa10a) [inline README.md]: Also added a new feature for running the server more easily

**Comment by circa10a** [comment]
The README kept giving conflict problems... nonetheless... this needs to be added to the readme.md

### Easy install

```
`bash -c "$(curl -sL https://raw.githubusercontent.com/nikssardana/gdriveTorrent/master/install.sh)"`
```

Note: (Tested on ubuntu)

apt-get/yum packages are not installed automatically due to security concerns, the installer will inform you of any packages missing (if there are any), but here is a list of packages needed:
```
sudo apt-get install python-pip python-dev build-essential aria2 git wget -y && sudo pip install virtualenv
```

After setup is complete, you may start the server at any time (or have a job do it) by running:

```
install.sh start_server
```

**Pull request merged**